src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,204 @@
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://mfreed7.github.io/backdrop-filter-feature/examples/scrollable.html" target="_blank">Demo</a>
+  </figcaption>
+</figure>
+
+### Text Contrast on Dynamic Backgrounds
+
+```css
+.background {
+  filter: invert(1);
+}
+
+.foreground {
+  backdrop-filter: invert(1);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-invert_color.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://tympanus.net/codrops-playground/huijing/Qqpwg5Iy/editor" target="_blank">Codrops</a>
+  </figcaption>
+</figure>
+
+### Censorship
+
+```css
+input:not(:checked) ~ .censor {
+  backdrop-filter: contrast(4) blur(20px);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-censorship.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://tympanus.net/codrops-playground/huijing/6cZVcORz/editor" target="_blank">Codrops</a>
+  </figcaption>
+</figure>
+
+
+## Conclusion
+Better late than never! Safari's had this feature since 2015. Regardless, Chrome's excited to release it in 76. The [Chromium bug](https://crbug.com/497522) has **560+ stars** on it, clearly marking it as a desired web design capability.
+
+Here's a final CSS API list and additional resources in case you're ready to start developing.
+
+### Overview
+```css
+/* Keyword value */
+backdrop-filter: none;
+
+/* URL to SVG filter */
+backdrop-filter: url(commonfilters.svg#filter);
+
+/* <filter-function> values */
+backdrop-filter: blur(2px);
+backdrop-filter: brightness(60%);
+backdrop-filter: contrast(40%);
+backdrop-filter: drop-shadow(4px 4px 10px blue);
+backdrop-filter: grayscale(30%);
+backdrop-filter: hue-rotate(120deg);
+backdrop-filter: invert(70%);
+backdrop-filter: opacity(20%);
+backdrop-filter: sepia(90%);
+backdrop-filter: saturate(80%);
+
+/* Multiple filters */
+backdrop-filter: url(filters.svg#filter) blur(4px) saturate(150%);
+
+/* Global values */
+backdrop-filter: inherit;
+backdrop-filter: initial;
+backdrop-filter: unset;
+```
+
+### One more demo
+A glimpse into the now possible in Chrome:
+
+<figure class="w-figure w-figure--fullbleed">
+  {% YouTube 'MWHBpReeAJ0' %}
+  <figcaption class="w-figcaption">
+    Demo by <a href="https://ferdychristant.com/please-help-make-backdrop-filter-a-reality-f81805ba3d52" target="_blank">Ferdy Christant</a>
+  </figcaption>
+</figure>
+
+### Additional Resources
+- [Specification](https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty)
+- [TAG Proposal](https://github.com/w3ctag/design-reviews/issues/353)
```

Comment:
I've never seen a TAG link in a web.dev or /Updates article. How is this useful for a web developer?

Reply:
i personally found it interesting and figured why put less resources? if people want to read more, this is a relevant item for that list. easily removed though, and if it's not normal, i'll pull it.